src/components/partials/tables/FTable.vue

```diff
@@ -0,0 +1,242 @@
+                    <slot name="control-right" />
+                </div>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col">
+
+                <q-tab-panels
+                    v-model="selectedTab"
+                    animated>
+                    <q-tab-panel
+                        v-for="tab in tabs"
+                        :key="tab.name"
+                        :name="tab.name"
+                        class="q-pa-none">
+                        <div class="row">
+                            <div class="col q-pa-md border-bottom-grey">
+                                <slot :name="'filter_' + tab.name"/>
+                            </div>
+                        </div>
+                        <slot :name="'table_' + tab.name"/>
+                    </q-tab-panel>
+
+                </q-tab-panels>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col">
+                <q-footer class="page-footer">
+                    <div
+                        class="q-px-md pagination">
+                        <slot name="paginate" />
+                    </div>
+                </q-footer>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+    import _ from 'lodash';
+
+    export default {
+        name: 'FTable',
+        props: {
+            pageHeader: {
+                type: Boolean,
+                default: true
+            },
+            tableTitle: {
+                type: String,
+                default: ''
+            },
+            TabsData: {
+                type: Array,
+                default: () => []
+            },
+            breadcrumbs: {
+                type: Array,
+                default: () => []
+            },
+            headerHelpText: {
+                type: String,
+                default: ''
+            },
+            defaultTab: {
+                type: String,
+                default: ''
+            },
+            tabIndicatorColor: {
+                type: String,
+                default: ''
+            }
+        },
+        data () {
+            return {
+                selectedTab: '',
+            };
+        },
+        computed: {
+            tabs() {
+                return this.TabsData.map(tab => {
+                    return {
+                        ...tab,
+                        label: this.$trans(_.snakeCase(tab.name)),
+                    };
+                });
+            },
+        },
+        created() {
+            this.selectedTab = this.defaultTab ? this.defaultTab : _.first(this.tabs).name;
+        },
+    };
+</script>
+
+<style scoped lang="stylus">
+    .border-bottom-grey
+        border-bottom: 1px solid $grey-5
+
+    .tab-content-class
+        font-weight 600
+
+    .pagination
+        background #FFFFFF 0% 0% no-repeat padding-box
+        box-shadow 0px -1px 3px #0000001A
+        border-radius 0px 0px 8px 8px
+        opacity 1
```

Comment:
Hi @axlepascual, @pandafox `<slot>` is already there as `control-left`, so we can use a text, a searchbar in some cases, dropdown buttons, etc. But to make it simpler for tabs I provided a way to pass tabs Data as props 
```
        <f-table
            :tabs-data="headerTabs"
            :selected-tab="selectedTab"
            :page-header="false"
            tab-indicator-color="transparent"
        >
```
So it will help to not write code of tabs repeatedly.

Reply:
thanks, updated.